src/BCAux.cxx

```diff
@@ -354,7 +354,7 @@ void BCAux::SetKnowledgeUpdateDrawingStyle(BCH2D& prior, BCH2D& posterior, BCAux
 
 
 // ---------------------------------------------------------
-void BCAux::DrawKnowledgeUpdate(BCHistogramBase& prior, BCHistogramBase& posterior, bool draw_prior_first)
+void BCAux::DrawKnowledgeUpdate(BCHistogramBase& prior, BCHistogramBase& posterior, bool draw_prior_first, BCTrash<TObject>& trash)
```

Comment:
If we want users to also use this code on their own, I think we need to have an example in the `examples` directory that shows the creation and use of a `BCTrash` object.

Reply:
You are right but I don't want users to use this on their own.